api/leaderboard/leaderboard_controller.py

```diff
@@ -27,6 +30,9 @@ async def get_leaderboard(self, page: int, limit: int, type: Optional[str] = Non
 
         return sorted_leaderboard[start_index:end_index]
 
+    async def get_leaderboard_size(self) -> int:
+        return await self._get_leaderboard_size()
+
```

Comment:
Ce que je voulais dire plus haut

```suggestion
        leaderboard = Leaderboard(sorted_leaderboard[start_index:end_index], self._get_leaderboard_size())
        
        return leaderboard
```

Reply:
Sure on peut mettre ça pour le get leaderboard, mais Caro avait besoin de la quantité total, pour le mettre le nombre de page dans la tableau, à moins qu'elle fait une requête attend le retour pour faire le reste de la page (ce qui n'est pas nécessairement mauvais). La seule chose que je suis moins sûr c'est si on a beacoup d'entré la page pourrait être plus long à loader si faut qu'on sort avant d'avoir le quantité total?

Reply:
Mais je ne suis pas contre de faire ça à place btw :)